compiler/rustc_hir_typeck/src/loops.rs

```diff
@@ -43,7 +42,7 @@ enum Context {
     /// E.g. `#[loop_match] loop { state = 'label: { /* ... */ } }`.
     LoopMatch {
         /// The label of the labeled block (not of the loop itself).
```

Comment:
nit: This still says "label", should it be the following instead?
```suggestion
        /// The destination pointing to the labeled block (not of the loop itself).
```